.rubocop.yml

```diff
@@ -30,3 +30,6 @@ Metrics/ClassLength:
 Performance/FixedSize:
   Exclude:
     - 'test/**/*'
+
+Style/AndOr:
+  EnforcedStyle: conditionals
```

Comment:
I think setting this to `always` is reasonable, as the vast majority of additional warnings it generates are actually for logic operations which should to be changed to `&&` and `||`.

There are quite a few uses of `and` in controllers for actual control flow (e.g. `redirect.. and return`), but I'm unconvinced that this is correct. It appears to be a shorthand for putting things onto a single line rather than performing control flow based on the return value of the first statement, so should be written as two lines.